### PR TITLE
Fix/ca bug

### PR DESCRIPTION
**Pull request by MiroslavRepka**
Fix the bug introduced in #805, with the incorrect tag when k8s version is not supporting external gRPC cloud provider.

### CODE REVIEW

**Review by cloudziu — APPROVED** [review]
LGTM 👍

**Review by cloudziu — APPROVED** [review]

**Review by cloudziu — APPROVED** [review]

File: manifests/claudie/kustomization.yaml [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
-- scheduler.yaml
-- builder.yaml
-- kube-eleven.yaml
-- frontend.yaml
-- kuber.yaml
-- minio
-- mongo
-- dynamo
-- network-policy.yaml
+  - ns.yaml
```
Comment (cloudziu) [inline manifests/claudie/kustomization.yaml]: ❤️

**Review by cloudziu — APPROVED** [review]
LGTM 👍

File: manifests/claudie/kustomization.yaml (hunk 1 above)
Comment (MiroslavRepka) [reply manifests/claudie/kustomization.yaml]: The `kustomize edit ` will revert it back anyway :smile:

Comment (cloudziu) [reply manifests/claudie/kustomization.yaml]: Really? When running locally, I did not have issues with it 🤔 Did it started working like this when I bumped the version to `4.5.6`?

**Pull request merged**